src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -45,7 +45,7 @@ DlgPrefSound::DlgPrefSound(QWidget* pParent, SoundManager* pSoundManager,
     connect(m_pSoundManager, &SoundManager::devicesUpdated, this, &DlgPrefSound::refreshDevices);
 
     apiComboBox->clear();
-    apiComboBox->addItem(tr("None"), "None");
+    apiComboBox->addItem(MIXXX_NOSOUNDAPI_STRING, tr("None"));
```

Comment:
Here "None" as user data is not translated, because it can than be identified in any language.

Reply:
fixed, it's just `SoundManagerConfig::kEmptyComboBox` now // " --- "